packages/web-media-box/src/pipelines/mse/mseManager.ts

```diff
@@ -0,0 +1,178 @@
+import Logger from "@/utils/logger";
```

Comment:
we are using Logger as a type here, so we can add `import type`

Reply:
fixed